### PR TITLE
PKIsigning platform (certified connector)

### PR DESCRIPTION
**Pull request by pkisigning**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Hi @pkisigning! Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

**Comment by pkisigning** [comment]
With all the respect to Srikanth, January 31st is more than two weeks ago now. Could you tell me anything about how much more delay we should be expecting? (@natalie-pienkowska) I would presume a platform like this to be operated by more than one certification engineer, or there should be some kind of procedure to back someone’s tasks when a person is unavailable for some time. It just doesn’t feel Microsoft alike to have radio silence for so long.

**Comment by Amjed-Ayoub** [comment]
@pkisigning Thank you for reaching out. Apologies for the delay in response. @sriyen-msft our Primary Engineer will review your request ASAP. Please expect to hear back from us very soon. Thank you for your patience.

File: certified-connectors/PKIsigning/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            "refreshUrl": {
+              "value": "https://identity.pkisigning.io/connect/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0E6EFE",
+    "capabilities": [],
+    "publisher": "Wim Zoet"
```
Comment (sriyen-msft) [inline certified-connectors/PKIsigning/apiProperties.json:38]: @pkisigning Please add stackOwner property. 

Docs - [Add Metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3-add-metadata)

Comment (sriyen-msft) [inline certified-connectors/PKIsigning/apiProperties.json:38]: @pkisigning The publisher name seems an individual name which is not allowed, It should be an organization or company name. 

Docs - [Add Metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties)

File: certified-connectors/PKIsigning/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2349 @@
...
+          "description": "Actors within the document",
+          "items": {
+            "$ref": "#/definitions/Signer"
+          }
+        }
+      }
+    },
+    "DocumentTypes": {
+      "type": "string",
+      "description": "",
```
Comment (sriyen-msft) [inline certified-connectors/PKIsigning/apiDefinition.swagger.json:1990]: @pkisigning Empty property values can be removed until unless you have a specific reason to keep them.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/PKIsigning/readme.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+
+## PKIsigning Platform Connector
+The PKIsigning platform provides an extensive REST API. Using this API, you can manage all files and their contained documents and requests for signing, approving and downloading files. 
+
```
[the remark this answers is not in the document]
Comment (pkisigning) [reply certified-connectors/PKIsigning/readme.md:4]: We might have used the wrong template. Template was verified against the template in the templates folder and missing information was added.

File: certified-connectors/PKIsigning/apiProperties.json (hunk 1 above)
Comment (pkisigning) [reply certified-connectors/PKIsigning/apiProperties.json]: stackOwner was added.

File: certified-connectors/PKIsigning/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "E7A018BA-2E96-4D90-9ADD-CF593DE86CD5",
```
[the remark this answers is not in the document]
Comment (pkisigning) [reply certified-connectors/PKIsigning/apiProperties.json]: value has been updated

File: certified-connectors/PKIsigning/apiProperties.json (hunk 1 above)
Comment (pkisigning) [reply certified-connectors/PKIsigning/apiProperties.json]: publisher has been updated to be equal to stackOwner property and references our company name. Please note PKIsigning is not our company name, but just our trademark. This is mentioned in readme.md as well.

File: certified-connectors/PKIsigning/apiDefinition.swagger.json (hunk 2 above)
Comment (pkisigning) [reply certified-connectors/PKIsigning/apiDefinition.swagger.json]: all empty descriptions now either have a value, or have been removed.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**